tuned/plugins/plugin_cpu.py

```diff
@@ -16,7 +16,131 @@
 
 class CPULatencyPlugin(base.Plugin):
 	"""
-	Plugin for tuning CPU options. Powersaving, governor, required latency, etc.
+	`cpu`::
+	
+	Sets the CPU governor to the value specified by the [option]`governor`
+	option and dynamically changes the Power Management Quality of
+	Service (PM QoS) CPU Direct Memory Access (DMA) latency according
+	to the CPU load.
+	
+	`governor`:::
+	The [option]`governor` option of the 'cpu' plug-in supports specifying
+	CPU governors. Multiple governors are separated using '|'. The '|'
+	character is meant to represent a logical 'or' operator. Note the
+	same syntax is used for the [option]`energy_perf_bias` option. *TuneD*
+	will set the first governor that is available on the system.
+	+    
+	For example, with the following profile, *TuneD* will set the 'ondemand'
+	governor, if it's available. If it's not available, but the 'powersave'
+	governor is available, 'powersave' will be set. If neither of them are
+	available, the governor will not be changed.
+	+
+	.Specifying a CPU governor
+	====
+	----
+	[cpu]
+	governor=ondemand|powersave
+	----
+	====
+	
+	`sampling_down_factor`:::
+	The sampling rate determines how frequently the governor checks
+	to tune the CPU. The [option]`sampling_down_factor` is a tunable
+	that multiplies the sampling rate when the CPU is at its highest
+	clock frequency thereby delaying load evaluation and improving
+	performance. Allowed values for sampling_down_factor are 1 to 100000.
+	+
+	.The recommended setting for jitter reduction
+	====
+	----
+	[cpu]
+	sampling_down_factor = 100
+	----
+	====
+	
+	`energy_perf_bias`:::
+	[option]`energy_perf_bias` supports managing energy
+	vs. performance policy via x86 Model Specific Registers using the
+	x86_energy_perf_policy tool. Multiple alternative Energy Performance
+	Bias (EPB) values are supported. The alternative values are separated
+	using the '|' character. The following EPB values are supported
+	starting with kernel 4.13: "performance", "balance-performance",
+	"normal", "balance-power" and "power".
+	+
+	.Specifying alternative Energy Performance Bias values
+	====
+	----
+	[cpu]
+	energy_perf_bias=powersave|power
+	----
+	
+	*TuneD* will try to set EPB to 'powersave' (pre kernel 4.13 value). If
+	that fails, it will try to set it to 'power'.
+	====
+	
+	`latency_low, latency_high, load_threshold`:::
+	+
+	If the CPU load is lower than the value specified by the
+	[option]`load_threshold` option, the latency is set to the value
+	specified by the [option]`latency_high` option, otherwise it is set
+	to the value specified by [option]`latency_low`.
+	+
```

Comment:
If the CPU load is lower than the value specified by the	[option]`load_threshold` option, the latency is set to the value specified either by the [option]`latency_high` option or by the [option]`latency_low` option.